R/run-nmtran.R

```diff
@@ -0,0 +1,223 @@
+#' Interface for running `NM-TRAN` on model objects
+#'
+#' Provides functions to run `NM-TRAN` on model objects for validation and to
+#' generate `NM-TRAN` datasets (`FDATA`).
+#'
+#' @details
+#' `NM-TRAN` is a preprocessor for `NONMEM` that translates user-specified
+#' control stream data and instructions into a form executable by `NONMEM`.
+#'
+#' `run_nmtran()` allows users to test their models ahead of submission to ensure
+#' correct coding.
+#'
+#' @section Supported `bbi_args`:
+#' `run_nmtran()` supports passing the following raw `NMFE` options through to `NM-TRAN`
+#' via the familiar `.bbi_args` argument.
+#'  - **`maxlim`**: Set the maximum values for the buffers used by `NONMEM`
+#'  (if 0, don't pass `-maxlim` to nmfe) (default is 2)
+#'    - If `maxlim = 3`, it is preferred to also set `tprdefault = TRUE`, but
+#'    _not_ `prdefault`, as `NM-TRAN`'s optional resizing of the `PREDPP` size
+#'    parameter `MAXRECID` may conflict with the `-prdefault` option.
+#'  - **`prdefault`**: If `TRUE`, do not recompile any routines other than `FSUBS`
+#'  - **`tprdefault`**: If `TRUE`, test if is okay to do `-prdefault`
```

Comment:
I think that we should drop this section and just point to `print_bbi_args` in the description of the `.bbi_args` parameter below.

Reply:
What do you think of this bullet though?
> - If `maxlim = 3`, it is preferred to also set `tprdefault = TRUE`, but
 _not_ `prdefault`, as `NM-TRAN`'s optional resizing of the `PREDPP` size

This is mentioned in the `NONMEM` docs, and I thought it might provide some additional value not highlighted in `BBI_ARGS`

Reply:
In my opinion this isn't the spot to try to relay info from the NONMEM docs.  If the caller is passing something other than the default here, they plan to do the same for submitting the model and already know what they're doing.